index.js

```diff
@@ -0,0 +1,98 @@
+import domSelectors from './modules/domSelectors.js';
+import Store from './modules/LocalStorage.js';
+import Book from './modules/Book.js';
+// import UI from './modules/UI.js';
+
+setInterval(() => {
+  document.getElementById('date').textContent = new Date();
+}, 1000);
```

Comment:
- [ ] (Required) For this milestone, you are required to use Luxon library as a javascript(.js) file and import the `DateTime` object to display the current date and time under the navbar. It means you would have to download the minified or full version of Luxon as a javascript file and use it. There are many docs that have download links to the Luxon Library when you search on Google, but to give you a head start, you can check out this [docs](https://www.cdnpkg.com/fullcalendar/file/luxon.min.js/). 👍

Reply:
I have added the Luxon Implementation in the project.